runtime/near-vm-logic/Cargo.toml

```diff
@@ -21,7 +21,9 @@ serde = { version = "1", features = ["derive"] }
 sha2 = ">=0.8,<0.10"
 sha3 = ">=0.8,<0.10"
 
-near-primitives = { path = "../../core/primitives" }
+# If you want to uncomment this, think twice: `near-primitives` are a heavy dependency,
+# we are intentionally avoiding it here.
```

Comment:
Maybe we want to bring more context here like here:

https://github.com/near/nearcore/blob/7e24d5d758be41ff12528964ebdcad12e32b990d/runtime/near-runtime-utils/Cargo.toml#L16-L17